internal/extproc/chatcompletion_processor.go

```diff
@@ -78,15 +86,18 @@ func (c *chatCompletionProcessor) ProcessRequestHeaders(_ context.Context, _ *co
 
 // ProcessRequestBody implements [Processor.ProcessRequestBody].
 func (c *chatCompletionProcessor) ProcessRequestBody(ctx context.Context, rawBody *extprocv3.HttpBody) (res *extprocv3.ProcessingResponse, err error) {
+	c.requestStart = time.Now() // Track the start time of the request
```

Comment:
Should we set the `requestStart` in the OnRequestHeaders method?

Reply:
i don't see this method yet

Reply:
Sorry, I meant `ProcessRequestHeaders`